Harvest.Web/ClientApp/src/Closeout/CloseoutContainer.tsx

```diff
@@ -0,0 +1,187 @@
+      method: "POST"
+    });
+
+    setNotification(request, "Closing Out Project", async (response: Response) => {
+      const result = await response.json() as Result<number>;
+      setCloseoutRequested(true);
+      return result.message;
+    });
+  }
+
+  if (project === undefined) {
+    return <div>Loading ...</div>;
+  }
+
+  const canCloseout = project.status === "AwaitingCloseout" ||
+    (project.status === "Active" && beyondCloseoutDisplayDate);
+
+  return (
+    <div className="card-wrapper">
+      <ProjectHeader
+        project={project}
+        title={"Field Request #" + (project.id || "")}
+      ></ProjectHeader>
+      <div className="card-green-bg">
+        <div className="card-content">
+          <div className="row">
+            <div className="col-md-6">
+              <h2>Prepare final invoice for project closeout</h2>
+              <div className="row">
+                <div className="col-md-8">
+                  <Label for="amount">Final Acreage Expense (defaults to monthly)</Label>
+                  <Input
+                    className={getClassName("amount")}
+                    type="number"
+                    id="amount"
+                    value={finalAcreageExpense.amount}
+                    onChange={onChange("amount", (e) => setFinalAcreageExpense({ amount: parseFloat(e.target.value) }))}
+                    onBlur={onBlur("amount")}
+                  />
+                  <InputErrorMessage name="amount" />
+                </div>
+                <div className="col-md-4">
+                  <Button color="primary" disabled={notification.pending || formErrorCount > 0} onClick={addAcreageExpense}>
+                    Add Acreage Expense <FontAwesomeIcon icon={faPlus} />
+                  </Button>
+                </div>
+
+              </div>
+            </div>
+            <div className="col-md-6">
+              <ProjectProgress project={project} />
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="card-content">
+        <div className="row">
+          <div className="col-md-12">
+            <UnbilledExpensesContainer newExpenseCount={newExpenseCount} />
+          </div>
+        </div>
+      </div>
+      <div className="card-content card-green-bg">
+        <div className="row">
+          <div className="col text-right">
+            <Button id="CloseoutButton" color="primary" disabled={!canCloseout || notification.pending} onClick={closeoutProject}>
+              Closeout Project <FontAwesomeIcon icon={faCheck} />
+            </Button>
+            <ShowFor condition={!canCloseout}>
+              {/* Not sure if there's a better way to handle indicators as to why something is disabled */}
+              <UncontrolledTooltip target="CloseoutButton" >
+                Project is not awaiting closeout, and it doesn't have an end date that falls within the last seven days or later.
```

Comment:
For my emails, I just picked 7 days.
I don't think we want to limit the closeout to that.